[[ Bug 21206 ]] Fix error opening msg box from SE

montegoulding opened the pull request:
This patch adds `actionLaunchMessageBox` to the script editor stack
behavior so that the Cmd+M shortcut does not cause an execution error.

livecodeali commented on Toolset/palettes/script editor/behaviors/revsestackbehavior.livecodescript:
```diff
@@ -2144,6 +2144,13 @@ command actionAutocompleteSnippetManager
 
+command actionLaunchMessageBox
+   local tStackName
+   put revIDEPaletteToStackName("Message Box") into tStackName
```
You don't need this part - revIDEOpenPalette maps to the stack name

montegoulding replied:
ok, done

montegoulding committed:
[[ Bug 21206 ]] Fix error opening msg box from SE

This patch adds `actionLaunchMessageBox` to the script editor stack
behavior so that the Cmd+M shortcut does not cause an execution error.

livecodeali commented:
@livecode-vulcan review ok 5c25591

livecode-vulcan commented:
💙 review by @livecodeali ok 5c25591dce8d8c1d0fdf804ec7f08ca5843d5592
